src/main/java/frc/robot/RobotPreferences.java

```diff
@@ -119,6 +119,35 @@ public static final class prefDrivetrain {
   }
 
   public static final class prefIntake {
+    public static final SN_DoublePreference intakePivotP = new SN_DoublePreference("ïntakePivotP", 24);
+    public static final SN_DoublePreference intakePivotI = new SN_DoublePreference("ïntakePivotI", 0);
+    public static final SN_DoublePreference intakePivotD = new SN_DoublePreference("ïntakePivotD", 0);
+    /**
+     * Enable or disable the intake pivot forward limit
+     */
+    public static final SN_BooleanPreference intakePivotForwardLimitEnable = new SN_BooleanPreference(
+        "intakePivotForwardLimitEnable", true);
+    /**
+     * <p>
+     * The maximum soft limit of the intake pivot
+     * </p>
+     * <b>Units:</b> Rotations
+     */
+    public static final SN_DoublePreference intakeForwardLimit = new SN_DoublePreference("intakeForwardLimit",
```

Comment:
Just to keep it consistent, can you call this (and line 149) intakePivotForwardLimit? just add the pivot word in there. Otherwise looks great